src/main/java/org/radarcns/management/domain/SourceType.java

```diff
@@ -55,10 +60,12 @@
     private String appProvider;
 
     @NotNull
+    @Pattern(regexp = Constants.ENTITY_ID_REGEX)
     @Column(name = "model", nullable = false )
     private String model;
 
     @NotNull
+    @Pattern(regexp = Constants.ENTITY_ID_REGEX)
```

Comment:
Should we add these `@pattern ` annotations to other resources as well, since we have a pattern requirement now.

Reply:
I added them to all fields that are part of the entity's identifier. E.g. `sourceName` for a `Source`, `projectName` for a `Project`, or `producer`, `model` and `catalogVersion` in the case of `SourceType`. Do you have other suggestions, or did i miss any?

Reply:
SourdeDataName for sourceData

Reply:
My bad. SourceDataName is there